tallysman:
  tallysman_gps:
    ros__parameters:
      save_logs: False
      log_level: 20

tallysman:                  
  gps_publisher:
    ros__parameters:
      save_logs: False
      log_level: 20

tallysman:
  base:
    ros__parameters:
      save_logs: False
      log_level: 20

tallysman:
  rover:
    ros__parameters:
      save_logs: False
      log_level: 20